hipcheck/src/plugin/types.rs

```diff
@@ -185,8 +186,9 @@ impl PluginContext {
 		);
 		self.set_configuration(&config).await?.as_result()?;
 		let default_policy_expr = self.get_default_policy_expression().await?;
-		let (tx, mut out_rx) = tokio::sync::mpsc::channel::<PluginQuery>(10);
+		let (tx, mut out_rx) = mpsc::channel::<PluginQuery>(10);
```

Comment:
In the future we'll want this queue size to be configurable.

Reply:
I'll make an issue